Andrea / exercise-1/stylesheet.css

```diff
@@ -0,0 +1,209 @@
+/* Universal */
+
+*{
+    margin :0;
+    padding : 0;
+    box-sizing : border-box;
+
+}
+
+main{
+    margin-bottom: 32px;
+}
+
+h3 {
+    font-family: "Monserrat", sans-serif;
+    font-size: 18px;
+    margin: 16px ;
+    text-transform: capitalize;
+}
+
+
+/* Header*/
+a{
+    font-family: "Monserrat", sans-serif;
+    color: black;
+    text-decoration: none;
+    text-transform: capitalize;
+}
+
+a img, li img{
+    width: 20px;
+    height: 20px;
+    margin-right: 16px;
+}
+
+li{
+    font-family: "Monserrat", sans-serif;
+    font-size: 16px;
+    list-style: none;
+    line-height: 2;
+    text-transform: capitalize;
+}
+
+span {
+    display: block;
+}
+
+p{
+    font-family: "Monserrat", sans-serif;
+    font-size: 16px;
+}
+
+button{
+    flex: 1;
+    font-family: "Monserrat", sans-serif;
+    font-size: 16px;
+    text-transform: capitalize;
+    border: 1px solid brown;
+    border-radius: 8px;
+    padding: 4px;
+    margin-right: 16px;
+} 
+
+
+button:hover{
+    background-color: rgb(214, 187, 176);  
+  }
+
+/* Especifico */
+.hero {
+    position: relative;
+    width: 100wv;
+    height: 100vh;
+    display: flex;
+    justify-content: center; /* Horizontalmente */
+    align-items: center; /* Verticalmente */
+    
+}
+
+.hero h1 {
+    position: relative;
+    color: white;
+    text-align: center;
+    font-family: "Montserrat", sans-serif;
+    font-size: 55px;
+    font-weight: normal;
+    text-transform: capitalize;
+}
+
+.hero::before {
+    content: "";
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background: url(hero.jpeg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center center;
+    filter: brightness(60%);
+}
+
+.section-menu {
+    margin: 8px 16px;
+}
+
+.info {
+  display: flex; /* The default is flex-direction:row; */
+  align-items: center;
+} 
+
+.info-img {
+    flex: 1;
+    align-content: center;
+}
```

Comment:
@frank-martinez-iest for the image in the main section, I used this to change the side if the screen is bigger, is it correct or is there another way for doing it?

Reply:
If you want to make it interesting, I challenge you to use `display: grid` in the `.info` class instead of using flex. Even though I would use flexbox for this particular scenario is a good use case to practice grid. You can follow this guid https://css-tricks.com/snippets/css/complete-guide-grid/

Reply:
@frank-martinez-iest I am using grid but I have a question: does this means that I no longer need to use flex inside the children of .info?
Like this:
`.info-img {
    flex: 1;
    align-content: center;
}`
I would not use this anymore since the grid does the work, right?